clang/test/CodeGenSYCL/reqd-work-group-size.cpp

```diff
@@ -17,40 +17,6 @@ class Functor32x16x16 {
 public:
   [[sycl::reqd_work_group_size(32, 16, 16)]] void operator()() const {}
 };
-class CLFunctor32x16x16 {
```

Comment:
Is [[cl::reqd_work_group_size( )]] not relevant anymore?

Reply:
We still need this spelling I believe. We have deprecated this in SYCL 2020 mode: [[cl::reqd_work_group_size]]

https://github.com/intel/llvm/pull/3689